qutebrowser/browser/webengine/webenginesettings.py

```diff
@@ -50,37 +59,41 @@ class _SettingsWrapper:
     For read operations, the default profile value is always used.
     """
 
-    def __init__(self):
-        self._settings = [default_profile.settings(),
-                          private_profile.settings()]
```

Comment:
We can't keep these references to the profiles' settings anymore in case the profiles change.

Reply:
That sounds good.